schema.yaml

```diff
@@ -0,0 +1,257 @@
+"$schema": https://json-schema.org/draft/2020-12/schema#
+title: CDBA Configuration Schema
+type: object
+properties:
+  devices:
+    type: array
+    items:
+      type: object
+      properties:
+        board:
+          type: string
+
+        name:
+          type: string
+
+        description:
+          type: string
+
+        console:
+          type: string
+          pattern: "^/dev/.*$"
+
+        fastboot:
+          type: string
+          pattern: "^[0-9a-f]{8}$"
+
+        fastboot_set_active:
+          type: boolean
+
+        broken_fastboot_boot:
+          type: boolean
+
+        usb_always_on:
+          type: boolean
+
+        fastboot_key_timeout:
+          type: integer
+          minimum: 1
+
+        cdba:
+          type: string
+          pattern: "^/dev/.*$"
+
+        conmux:
+          type: string
+          pattern: "^/dev/.*$"
```

Comment:
Huh? No, `conmux` is a service string, not a device.

Reply:
Ack, thx for confirming